### ISSUE TITLE
feedburner.com

### ISSUE THREAD

**Issue by xxcriticxx** [mitchellkrogza/Ultimate.Hosts.Blacklist#455]
Match found in https://hosts.ubuntu101.co.za/hosts:
   feedburner.com

**Comment by funilrys** [comment]
Closing was a mistake.

**Comment by dnmTX** [comment]
It redirects to `accounts.google.com`.I think it's safe to whitelist.

**Comment by funilrys** [comment]
Actually:
```
> GET / HTTP/1.1
> Host: feedburner.com
> GET /fb/a/home HTTP/1.1
> Host: feedburner.com
> GET /accounts/ServiceLogin?service=feedburner&continue=https%3A%2F%2Ffeedburner.google.com%2Ffb%2Fa%2Fmyfeeds&gsessionid=I7r4xzcE7cQTUWOvl8i1mgWD9PA0gGkM HTTP/2
> Host: www.google.com
> GET /ServiceLogin?service=feedburner&continue=https%3A%2F%2Ffeedburner.google.com%2Ffb%2Fa%2Fmyfeeds&gsessionid=I7r4xzcE7cQTUWOvl8i1mgWD9PA0gGkM HTTP/2
> Host: accounts.google.com
```

**Comment by dnmTX** [comment]
@funilrys  it sure is go first to `accounts.google.com` and after one logged in it ends up in `feedburner.google.com`. Tested it myself. Would advise you to wait on @anudeepND for this one.

**Comment by funilrys** [comment]
No worry if it is on hold for so long there is a reason ...

**Comment by spirillen** [comment]
Hi guys @anudeepND have posted hit answer here https://github.com/anudeepND/blacklist/issues/41 :) which makes my opinion to keep it in the blocklist
